packages/orbit-components/src/Truncate/Truncate.ct.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from "react";
+import { test, expect } from "@playwright/experimental-ct-react";
+
+import Heading from "../Heading";
+import Text from "../Text";
+import Stack from "../Stack";
+
+import Truncate from ".";
+
+test.describe("visual", () => {
```

Comment:
Can someone help me add the linux screenshots? I am running Docker with Colima and have problems running `yarn` and playwright.

Reply:
What exactly the problem is? 🤔

Reply:
I had only problem with docker itself, it was saying: "is docker engine running?" smth like that

Reply:
Running `yarn install --frozen-lockfile` inside Docker takes forever and ends up failing.

Reply:
```console
...
[5/6] Building fresh packages...
[1/18] ⠈ @parcel/watcher
[-/18] ⠈ waiting...
[8/18] ⠈ cypress
[9/18] ⠈ sharp
error /work/node_modules/@parcel/watcher: Command failed.
Exit code: 1
Command: node-gyp-build
Arguments: 
Directory: /work/node_modules/@parcel/watcher
Output:
gyp info it worked if it ends with ok
gyp info using node-gyp@9.4.0
gyp info using node@18.18.1 | linux | arm64
gyp info find Python using Python version 3.10.12 found at "/usr/bin/python3"
gyp http GET https://nodejs.org/download/release/v18.18.1/node-v18.18.1-headers.tar.gz
gyp http 200 https://nodejs.org/download/release/v18.18.1/node-v18.18.1-headers.tar.gz
gyp http GET https://nodejs.org/download/release/v18.18.1/SHASUMS256.txt
gyp http 200 https://nodejs.org/download/release/v18.18.1/SHASUMS256.txt
gyp info spawn /usr/bin/python3
gyp info spawn args [
gyp info spawn args   '/work/node_modules/node-gyp/gyp/gyp_main.py',
gyp info spawn args   'binding.gyp',
gyp info spawn args   '-f',
gyp info spawn args   'make',
gyp info spawn args   '-I',
gyp info spawn args   '/work/node_modules/@parcel/watcher/build/config.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/work/node_modules/node-gyp/addon.gypi',
gyp info spawn args   '-I',
gyp info spawn args   '/root/.cache/node-gyp/18.18.1/include/node/common.gypi',
gyp info spawn args   '-Dlibrary=shared_library',
gyp info spawn args   '-Dvisibility=default',
gyp info spawn args   '-Dnode_root_dir=/root/.cache/node-gyp/18.18.1',
gyp info spawn args   '-Dnode_gyp_dir=/work/node_modules/node-gyp',
gyp info spawn args   '-Dnode_lib_file=/root/.cache/node-gyp/18.18.1/<(target_arch)/node.lib',
gyp info spawn args   '-Dmodule_root_dir=/work/node_modules/@parcel/watcher',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp ERR! build error 
gyp ERR! stack Error: not found: make
gyp ERR! stack     at getNotFoundError (/work/node_modules/node-gyp/node_modules/which/which.js:10:17)
gyp ERR! stack     at /work/node_modules/node-gyp/node_modules/which/which.js:57:18
gyp ERR! stack     at new Promise (<anonymous>)
gyp ERR! stack     at step (/work/node_modules/node-gyp/node_modules/which/which.js:54:21)
gyp ERR! stack     at /work/node_modules/node-gyp/node_modules/which/which.js:71:22
gyp ERR! stack     at new Promise (<anonymous>)
gyp ERR! stack     at subStep (/work/node_modules/node-gyp/node_modules/which/which.js:69:33)
gyp ERR! stack     at /work/node_modules/node-gyp/node_modules/which/which.js:80:22
gyp ERR! stack     at /work/node_modules/isexe/index.js:42:5
gyp ERR! stack     at /work/node_modules/isexe/mode.js:8:5
gyp ERR! System Linux 5.10.109-0-virt
gyp ERR! command "/usr/bin/node" "/work/node_modules/.bin/node-gyp" "rebuild"
gyp ERR! cwd /work/node_modules/@parcel/watcher
```

Reply:
ignore the error, I get the same one

Reply:
For the record, as discussed offline, I get this output when I run `yarn components test-ct --update-snapshots` inside Docker with Colima.

```console
4) [Tablet] › Truncate/Truncate.ct.tsx:11:7 › visual › screenshot ────────────────────────────────

    Error: browserType.launch: Browser closed.
    ==================== Browser output: ====================
    <launching> /ms-playwright/webkit-1921/pw_run.sh --inspector-pipe --headless --no-startup-window
    <launched> pid=16715
    [pid=16715][err] 
    [pid=16715][err] (MiniBrowser:16721): GStreamer-WARNING **: 13:54:26.427: External plugin loader failed. This most likely means that the plugin loader helper binary was not found or could not be run. You might need to set the GST_PLUGIN_SCANNER environment variable if your setup is unusual. This should normally not be required though.
    [pid=16715][err] 
    [pid=16715][err] (MiniBrowser:16721): libsoup-ERROR **: 13:54:26.550: libsoup3 symbols detected. Using libsoup2 and libsoup3 in the same process is not supported.
    [pid=16715][err] /ms-playwright/webkit-1921/pw_run.sh: line 24: 16721 Trace/breakpoint trap   WEBKIT_FORCE_COMPLEX_TEXT="1" "$MINIBROWSER" "$@"
    =========================== logs ===========================
    <launching> /ms-playwright/webkit-1921/pw_run.sh --inspector-pipe --headless --no-startup-window
    <launched> pid=16715
    [pid=16715][err] 
    [pid=16715][err] (MiniBrowser:16721): GStreamer-WARNING **: 13:54:26.427: External plugin loader failed. This most likely means that the plugin loader helper binary was not found or could not be run. You might need to set the GST_PLUGIN_SCANNER environment variable if your setup is unusual. This should normally not be required though.
    [pid=16715][err] 
    [pid=16715][err] (MiniBrowser:16721): libsoup-ERROR **: 13:54:26.550: libsoup3 symbols detected. Using libsoup2 and libsoup3 in the same process is not supported.
    [pid=16715][err] /ms-playwright/webkit-1921/pw_run.sh: line 24: 16721 Trace/breakpoint trap   WEBKIT_FORCE_COMPLEX_TEXT="1" "$MINIBROWSER" "$@"
    ============================================================
```

As we're not sure if the problem is related to Colima or architecture, I've removed the commit adding component tests.